cylc/flow/taskdef.py

```diff
@@ -85,11 +92,7 @@ def generate_graph_parents(tdef, point):
             if is_abs and parent_point != point:
                 # If 'foo[^] => bar' only spawn off of '^'.
                 continue
-            if seq.is_valid(parent_point):
```

Comment:
The bug was use `parent_point` here instead of child `point`.  So can move the test up higher, should be a small efficiency improvement.